hail/python/hailtop/utils/utils.py

```diff
@@ -136,7 +136,7 @@ def generator():
     return generator()
 
 
-def unzip(lst):
+def unzip(lst: List[Any]) -> Tuple[Any]:
```

Comment:
So the way Tuple types work is if they return a tuple of two integers, then the type is `Tuple[int, int]`. If it returns a tuple of an integer and a string, then it would be `Tuple[int, str]`. So what should `Tuple[Any]` be? Take a look at line 145. What is the type of `a`? What is the type of `b`?

Hint: `lst` should have the type `Iterable[Tuple[Any, Any]]`. You can see this is true because of line 142. `for x, y in lst:`

Reply:
Should the return type then be `Tuple[any, Any] since it returns a tuple that can be either integers or string.

Reply:
What is the type of `a`? What is the type of `b`? You're returning a tuple that consists of the values `a` and `b`. Once you know the types of `a` and `b` then you can fill in what the return type should be.

Reply:
`a` and `b` are both list that contain either Strings or Integers

Reply:
Great. So `Any` is shorthand for any possible type. Since the lists `a` and `b` can contain any element type (strings, integers, floats, booleans, and even more complex types like lists, tuples, dictionaries), then we use the type `Any` to allow any possible type.

This is why the type of `lst` is `Iterable[Any]`.